src/Cms/App.php

```diff
@@ -365,16 +365,17 @@ public function clone(array $props = [], bool $setInstance = true)
 	 * automatically injected
 	 *
 	 * @param string $name
+	 * @param string $options
 	 * @return \Kirby\Cms\Collection|null
 	 */
-	public function collection(string $name)
+	public function collection(string $name, array $options = [])
 	{
-		return $this->collections()->get($name, [
+		return $this->collections()->get($name, array_merge([
 			'kirby' => $this,
 			'site'  => $this->site(),
 			'pages' => $this->site()->children(),
 			'users' => $this->users()
-		]);
+		]), $options);
```

Comment:
I think the merge order should be the other way around so that the core objects always override the passed options. This ensures that the core objects can always be relied on.